### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by plimbu**
not my best work, text will re-arrange when slimming down browser

### CODE REVIEW

File: responsiveness-homework-starter-code/css/index.css [hunk 1]
```diff
@@ -33,16 +35,38 @@ header, .experience {
...
 	height: auto;
 	border: 1px solid #E13A63;
 	display: inline-block;
+
 }
-.containerimg
+
+
+.containertext1
 {
```
Comment (JSChilds) [inline responsiveness-homework-starter-code/css/index.css:47]: This is not a very descriptive class name. Also, you apply this class to multiple elements, whilst the name implies it just refers to one element. Be more specific with your class names when possible.

File: responsiveness-homework-starter-code/css/index.css [hunk 2]
```diff
@@ -33,16 +35,38 @@ header, .experience {
...
-	width: auto;
 	height: auto;
 	border: 1px solid #E13A63;
 	display: inline-block;
 
 }
+
+
+.containerimg
+{
```
Comment (JSChilds) [inline responsiveness-homework-starter-code/css/index.css:56]: Be consistent about how you write your code. Here, you write your css like this

.class-name
{
  attribute: value;
}

Whereas above, you write it like this:

.class-name {
  attribute: value;
}

Both are fine and there's not a huge difference, but it's distracting if things aren't consistent.

File: responsiveness-homework-starter-code/index.html [hunk 3]
```diff
@@ -32,26 +32,29 @@
...
-        <div  class="containertext">
-          <p>University of Portsmouth</p>
-          <p>BSc Economics, Finance and Banking 2:1</p>
-        </div>
+
+
 
         <div class="containertext">
+
+                  <div  >
```
Comment (JSChilds) [inline responsiveness-homework-starter-code/index.html:45]: this should be <div>, not <div > with a space.

File: responsiveness-homework-starter-code/index.html [hunk 4]
```diff
@@ -32,26 +32,29 @@
...
+                    <h1>Stephen Reid</h1>
```
Comment (JSChilds) [inline responsiveness-homework-starter-code/index.html:46]: This is too far indented.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Good effort, Peter. The responsiveness works well and you've made some nice styling choices. The columns with the screenshots in the projects page still need work and I would also urge you to maintain consistency with your styling, i.e. - the index page has a border around the headshot and profile summary, but the projects page doesn't. Try to keep styling consistent across your pages whenever possible.

File: responsiveness-homework-starter-code/index.html (hunk 3 above)
Comment (A-lina) [reply responsiveness-homework-starter-code/index.html:45]: It was about div tag (there shouldn't be any spaces).

**Pull request merged**